R/methods-epi_archive.R

```diff
@@ -360,6 +352,224 @@ epix_merge = function(x, y,
   ))
 }
 
+#' combine epi_archives by rows
+#'
+#' Take a sequence of archives and combine by rows. Complications arise if
+#' there are `time_value`s shared between the lists. `sync` determines how
+#' any later `NA`'s are treated, with the default `"forbid"` throwing an error,
+#' `"na"` treating them as intentional data (no modification), and `"locf"`
+#' filling forward across versions.
+#' Shared keys are another problem; by default, `force_distinct=FALSE`, meaning
+#' the entry in the earlier archive overwrites later archives. Otherwise there
+#' is an error on shared keys
+#' this function is still under active development, so there may be remaining
+#' edge cases
+#'
+#' @param ... list of `epi_archive` objects to append in order.
+#' @param sync Optional; `"forbid"`, `"na"`, or `"locf"`; in the case that later
```

Comment:
Yeah, tracks. I'd rather not use `fill` directly since I have to call that function; maybe `fill_na`, since it only effects `NA`s? I think `clobber` fits better with `force_distinct`, and will probably be getting its own parameter based on other discussions. `"keep"` is definitely the right word for what's currently `"na"`